.github/workflows/canary-integration-test-arm64.yml

```diff
@@ -0,0 +1,74 @@
+name: Canary integration tests ARM64
+on:
+  schedule:
+    - cron: '0 0 * * *'  # every day at midnight
+
+jobs:
+  canary-arm64:
+    runs-on: [self-hosted, ubuntu-20.04, ARM64]
+    env:
+      BLOCK: /dev/sdb
+
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+
+    - name: setup golang
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.16
+
+    - name: teardown minikube and docker
+      run: |
+        uptime
+        minikube delete
+        docker system prune -a
+
+    - name: setup minikube
+      run: |
+        # sudo apt-get install build-essential -y
```

Comment:
Shall we remove all these commands that are commented?

Reply:
I decided to keep them if the runner goes away, so we have the entire procedure to set it up.